.changeset/cuddly-roses-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-site": major
+---
+
+Add createGraphQLClient into library, also add graphql-request as peerDependency
```

Comment:
```suggestion
Add `createGraphQLClient`

It can be used as a drop-in replacement for the client previously defined in the application code.
Adds [graphql-request](https://www.npmjs.com/package/graphql-request) as a peer dependency.
```